worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/MapPartitionFileWriter.java

```diff
@@ -197,17 +191,29 @@ public synchronized void destroy(IOException ioException) {
   }
 
   public void pushDataHandShake(int numReducePartitions, int bufferSize) {
+    logger.debug(
+        "FileWriter pushDataHandShake numReducePartitions:{} bufferSize:{}",
+        numReducePartitions,
+        bufferSize);
+
     this.numReducePartitions = numReducePartitions;
     fileInfo.setBufferSize(bufferSize);
+    fileInfo.setNumReducerPartitions(numReducePartitions);
   }
 
   public void regionStart(int currentDataRegionIndex, boolean isBroadcastRegion) {
+    logger.debug(
```

Comment:
It would be useful if we logger with fileinfo such as file pathname with this.

Reply:
ok, is's useful to mark which file it write